presto-phoenix/src/test/java/io/prestosql/plugin/phoenix/PhoenixQueryRunner.java

```diff
@@ -0,0 +1,154 @@
+        }
+
+        return queryRunner;
+    }
+
+    public static void createSchema(TestingPhoenixServer phoenixServer, String schema)
+            throws SQLException
+    {
+        try (Connection connection = DriverManager.getConnection(phoenixServer.getJdbcUrl());
+                Statement statement = connection.createStatement()) {
+            statement.execute(format("CREATE SCHEMA %s", schema));
+        }
+        catch (Exception e) {
+            throw new IllegalStateException("Can't create schema: " + schema, e);
+        }
+    }
+
+    private static void copyTpchTables(
+            QueryRunner queryRunner,
+            String sourceCatalog,
+            String sourceSchema,
+            Session session,
+            Iterable<TpchTable<?>> tables)
+    {
+        LOG.info("Loading data from %s.%s...", sourceCatalog, sourceSchema);
+        long startTime = System.nanoTime();
+        for (TpchTable<?> table : tables) {
+            copyTable(queryRunner, sourceCatalog, session, sourceSchema, table);
+        }
+        LOG.info("Loading from %s.%s complete in %s", sourceCatalog, sourceSchema, nanosSince(startTime).toString(SECONDS));
+    }
+
+    private static void copyTable(
+            QueryRunner queryRunner,
+            String catalog,
+            Session session,
+            String schema,
+            TpchTable<?> table)
+    {
+        QualifiedObjectName source = new QualifiedObjectName(catalog, schema, table.getTableName());
+        String target = table.getTableName();
+
+        @Language("SQL")
+        String sql;
+        switch (target) {
+            case "customer":
+                sql = format("CREATE TABLE %s AS SELECT * FROM %s", target, source);
+                break;
+            case "lineitem":
+                sql = format("CREATE TABLE %s WITH (ROWKEYS = 'ORDERKEY,LINENUMBER', SALT_BUCKETS=10) AS SELECT * FROM %s", target, source);
+                break;
+            case "orders":
+                sql = format("CREATE TABLE %s WITH (SALT_BUCKETS=10) AS SELECT * FROM %s", target, source);
+                break;
+            case "part":
+                sql = format("CREATE TABLE %s AS SELECT * FROM %s", target, source);
+                break;
+            case "partsupp":
+                sql = format("CREATE TABLE %s WITH (ROWKEYS = 'PARTKEY,SUPPKEY') AS SELECT * FROM %s", target, source);
+                break;
+            case "supplier":
+                sql = format("CREATE TABLE %s AS SELECT * FROM %s", target, source);
+                break;
+            default:
+                sql = format("CREATE TABLE %s AS SELECT * FROM %s", target, source);
+                break;
+        }
+
+        LOG.info("Running import for %s", target, sql);
+        LOG.info("%s", sql);
+        long start = System.nanoTime();
+        long rows = queryRunner.execute(session, sql).getUpdateCount().getAsLong();
+        LOG.info("Imported %s rows for %s in %s", rows, target, nanosSince(start));
```

Comment:
We should leverage Presto execution statistics for this. I see no point in logging this.